osf/utils/auth.py

```diff
@@ -30,27 +30,27 @@ def get_user(email=None, password=None, verification_key=None):
     query_list = []
     if email:
         email = email.strip().lower()
-        query_list.append(Q('emails__address', 'eq', email) | Q('username', 'eq', email))
+        query_list.append(Q(emails__address=email) | Q(username=email))
     if password:
         password = password.strip()
         try:
             query = query_list[0]
             for query_part in query_list[1:]:
```

Comment:
Am I correct in that `query_list` could only contain one query at this point (the `Q` from `if email:`) and this for loop can be removed?